template.go

```diff
@@ -337,12 +337,37 @@ func writeDebianGbpConf(dir string, dep14, pristineTar bool) error {
 
 	fmt.Fprintf(f, "[DEFAULT]\n")
 	if dep14 {
-		fmt.Fprintf(f, "debian-branch = debian/sid\n")
+		fmt.Fprintf(f, "debian-branch = debian/latest\n")
+		fmt.Fprintf(f, "upstream-branch = upstream/latest\n")
 		fmt.Fprintf(f, "dist = DEP14\n")
 	}
 	if pristineTar {
 		fmt.Fprintf(f, "pristine-tar = True\n")
 	}
+
+	// Additional text to the template which is useful for 99% of the go packages
+	// NOTE: The v%%(version%%~%%.)s will print v%(version%~%.)s
+	fmt.Fprintf(f, `
```

Comment:
As the format does not use any variable, [`fmt.Fprint`](https://pkg.go.dev/fmt#Fprint) should be used. This would allow to get rid of the `%` sign escaping.

Reply:
Right, why didn't I think about it, thanks for pointing out.